### PR TITLE
Add onboarding timeline for Clubs

### PR DESCRIPTION
**Pull request by bacharakis**

### CODE REVIEW

File: _pages/onboarding.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+{: .clear}
+&nbsp;
+
+
+
+<section class="timeline">
+  <ul>
+    <li>
+      <div>
+      Apply to Start or Transition a Club.
```
Comment (comzeradd) [inline _pages/onboarding.md]: you missed indentation here

File: static/css/app.css [hunk 2]
```diff
@@ -399,3 +399,63 @@ img.text-center {
...
+}
+
+.timeline ul li:nth-child(even) div {
+  left: -439px;
+}
+
+.timeline ul li:nth-child(even) div::before {
+  right: -15px;
+  border-width: 8px 0 8px 16px;
+  border-color: transparent transparent transparent #FFD75E;
```
Comment (comzeradd) [inline static/css/app.css]: Just to keep a unified indentation style, could you please use 4 space on all css rules?

**Review by comzeradd — CHANGES_REQUESTED** [review]
Nice work! :beers:
I really love the design of the page.

I left some comments for some minor issues.

On mobile I'd suggest we just stack the yellow cards, but we can do this in a later PR.

**Comment by bacharakis** [comment]
Also just fixed an issue raised in the navbar due to the extra menu item.

File: _pages/onboarding.md (hunk 1 above)
Comment (bacharakis) [reply _pages/onboarding.md]: Fixed in    73eb263

File: static/css/app.css (hunk 2 above)
Comment (bacharakis) [reply static/css/app.css]: fixed in    ca94eac

**Review by comzeradd — APPROVED** [review]

**Pull request merged**